### ISSUE TITLE
The brightness is reset to 50% when opening the tray icon again

### ISSUE THREAD

**Issue by yangdongdong333** [xanderfrangos/twinkle-tray#243]
After changing  brightness by click tray icon,the brightness is reset to 50% when opening the tray icon again.

![QQ截图20210923022837](https://user-images.githubusercontent.com/49440819/134403472-d04c6cec-1162-4e68-8ade-0cf06f732f7f.png)

And the brightness number will never change which is on  red line 

![QQ截图20210923024148](https://user-images.githubusercontent.com/49440819/134402851-cade2d47-dff7-4db9-b820-13ed20e3ff6c.png)

This picture show that twinkle-tray can't read the current brightness of monitor named VX2478-2.

In short, thank you. This software solves my inconvenience about monitor brightness adjustment。

**Comment by Ciancy28** [comment]
Same happens on my BenQ GL2250H
